scripts/lib/cleanup.sh

```diff
@@ -6,19 +6,21 @@ function check_path_cleanup(){
     do
         export NODE=$node
         envsubst '$NODE' < ${DIR}/test/check-cleanup-pod.yaml > ${DIR}/test/check-cleanup-pod-$node.yaml
-        kubectl apply -f ${DIR}/test/check-cleanup-pod-$node.yaml
+        kubectl apply -f ${DIR}/test/check-cleanup-pod-$node.yaml -n default
         rm -rf ${DIR}/test/check-cleanup-pod-$node.yaml
     done
     sleep 20
 
     for node in $worker_nodes
     do
-        if [[ $(kubectl get pods $node -ojsonpath="{.status.phase}") == "Failed" ]]; then
-            echo "BPF files not cleaned up on $node.. $(kubectl logs $node)"
-            exit 1
+        if [[ $(kubectl get pods -n default $node -ojsonpath="{.status.phase}") == "Failed" ]]; then
+            echo "BPF files not cleaned up on $node"
+            kubectl logs $node -n default
+            TEST_FAILED=true
+        else
+            echo "BPF files were cleaned up from the node $node"
         fi
-        kubectl delete pods $node
+        kubectl delete pods $node -n default
```

Comment:
Btw, we tear down the nodes so it is fine, if not cyclonus last test will leak some resources such as pods and ns..

Reply:
We use the `--cleanup-namespaces=true` in cyclonus tests so ideally if the job finishes, it ends up cleaning the namespaces that it had created.